tests/regression/test_interpolate_cross_mesh.py

```diff
@@ -215,7 +215,7 @@ def parameters(request):
         expected = np.asarray(
             [np.outer(coords[i], coords[i]) for i in range(len(coords))]
         )
-        V_src = FunctionSpace(m_src, "Regge", 2)  # Not point evaluation nodes
+        V_src = FunctionSpace(m_src, "Regge", 2, variant="point")  # Not point evaluation nodes
```

Comment:
kwarg and comment seem mismatched?

Reply:
The comment was already there. "Not point evaluation nodes" here refers to component point evaluation.

Reply:
Thanks for updating.